regression/contracts/assigns_enforce_arrays_12/test.desc

```diff
@@ -0,0 +1,10 @@
+CORE
+main.c
+--enforce-all-contracts
+^EXIT=10$
+^SIGNAL=0$
+^VERIFICATION FAILED$
+--
+--
+Checks whether verification fails when a function with an array assigns target
```

Comment:
Is there any warning printed for this case?

If yes, does it make sense to pattern match for it?

Reply:
We currently do not have any warnings related to this issue. Should I create a GitHub issue to add this in future PRs?